src/components/Chatbot/Chatbot.tsx

```diff
@@ -244,26 +251,26 @@ export default function HungerMapChatbot() {
                     chats[currentChatIndex].messages.map((message, index) => (
                       <div
                         key={index}
-                        className={`flex ${message.sender === 'user' ? 'justify-end' : 'justify-start'} mb-4`}
+                        className={`flex ${message.role === 'user' ? 'justify-end' : 'justify-start'} mb-4`}
                       >
-                        {message.sender === 'ai' && (
+                        {message.role === 'assistant' && (
                           <div className="relative chat-ai-message">
                             <Bot className={`w-6 h-6 fill-current ${isDarkMode ? 'text-white' : 'text-black'} `} />
                           </div>
                         )}
                         <div
                           className={`${
-                            message.sender === 'user'
+                            message.role === 'user'
                               ? isDarkMode
                                 ? 'chat-user-message-dark'
                                 : 'chat-user-message-light'
                               : 'chat-ai-message'
-                          } rounded-lg p-3 max-w-[80%] ${message.sender === 'user' ? 'ml-12' : ''}`}
+                          } rounded-lg p-3 max-w-[80%] ${message.role === 'user' ? 'ml-12' : ''}`}
```

Comment:
I know this is merged, but next time take a look at [clsx](https://www.npmjs.com/package/clsx)

Reply:
Nice tip, I didn't know clsx before!